plugins/javadoc/src/main/kotlin/javadoc/transformers/documentables/JavadocDocumentableSourceSetFilterTransformer.kt

```diff
@@ -0,0 +1,16 @@
+package javadoc.transformers.documentables
+
+import org.jetbrains.dokka.Platform
+import org.jetbrains.dokka.model.DModule
+import org.jetbrains.dokka.model.Documentable
+import org.jetbrains.dokka.transformers.documentation.PreMergeDocumentableTransformer
+
+class JavadocDocumentableSourceSetFilterTransformer : PreMergeDocumentableTransformer {
+    private val List<Documentable>.jvmSourceSets
+        get() = flatMap { it.sourceSets }.filter { it.analysisPlatform == Platform.jvm }.distinct()
```

Comment:
I'd personally also leave the `common` source sets that those jvm source sets depend upon

Reply:
uh ok, that means that i need to also figure out how to merge those sourcesets to ensure javadoc only works on 1 of them

Reply:
Why, can't we just use the Documentable merger as always? 

Also, I'd log an info that some source sets were dropped

Reply:
Bump